packages/seacas/libraries/ioss/src/catalyst/Iocatalyst_DatabaseIO.C

```diff
@@ -846,7 +897,7 @@ namespace Iocatalyst {
       this->readProperties(child[detail::PROPERTIES], region);
 
       // read fields (meta-data only)
-      this->readFields(child[detail::FIELDS], region);
+      this->readFields(child[detail::FIELDS], region, region->get_database());
```

Comment:
Similar to above comment, could get this argument internal to the `readFields` function instead of passing it in here.  Would involve same function call, but just inside function instead of at each call site.

Reply:
Good suggestion, I'll implement those changes and commit to this pull request!

Reply:
Done; rdy to merge